src/gui/views.py

```diff
@@ -325,27 +331,55 @@ def switch_to_roster_and_team_panes(self, event: UIEvent = None):
         self.command_bar_section.flush()
         self.command_bar_section.manager.enable()
 
-    def recruit_button(self) -> UIFlatButton:
+    def recruit_button(self) -> UITextureButton:
         """Attached Handler will change from displaying the roster & team panes
         to showing recruits available for hire, with the appropriate command bar.
 
         Returns:
             UIFlatButton: Button with attached handler.
         """
-        btn = UIFlatButton(
-            text="Recruit "
+        main_tex = PixelatedNinePatch(
+            left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[7]
+        )
+        pressed_tex = PixelatedNinePatch(
+            left=1, right=1, bottom=3, top=3, texture=WindowData.buttons[9]
+        )
+        hovered_tex = PixelatedNinePatch(
+            left=1, right=1, bottom=3, top=1, texture=WindowData.buttons[11]
+        )

```

Comment:
Seems like we might be in need of a module to allow this to be defined once and then shared amongst various places.